install.php

```diff
@@ -109,19 +109,24 @@
 
 	if (empty($message))
 	{
-		//Create config
+		$fullname = $_POST['surname'] . " " . $_POST['name'];
+		//Create config & privacy policy
 		$config = file_get_contents("config.php.template");
+		$policy = file_get_contents("policy.php");
 		$config = str_replace("##name##", $_POST['servername'], $config);
 		$config = str_replace("##title##", $_POST['title'], $config);
 		$config = str_replace("##url##", $_POST['url'], $config);
-		$config = str_replace("##mailer##", $_POST['mailer'], $config);
+		$policy = str_replace("##email##", $_POST['mailer_email'], $policy);
 		$config = str_replace("##mailer_email##", $_POST['mailer_email'], $config);
 		$config = str_replace("##server##", $_POST['server'], $config);
 		$config = str_replace("##database##", $_POST['database'], $config);
 		$config = str_replace("##user##", $_POST['dbuser'], $config);
 		$config = str_replace("##password##", $_POST['dbpassword'], $config);
 		$config = str_replace("##name##", $_POST['servername'], $config);
+		$policy = str_replace("##name##", $fullname, $policy);
 		file_put_contents("config.php", $config);
+		file_put_contents("policy.php", $policy);
```

Comment:
I would like lot more to instead use the constants defined in config.php to do this on runtime.

Reply:
Ah yes, my fault